cmd/issue_edit.go

```diff
@@ -54,6 +55,34 @@ var issueEditCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
+		deleteNote, err := cmd.Flags().GetBool("deletenote")
```

Comment:
I had the same impression while reviewing, but at the end of it I wished we could just use `--delete` but I'm afraid it might be ambiguous to other notes fields(instead of the note itself).

Reply:
Oh good idea.  I've changed this to 'delete-note'.